examples/spark-sensors/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 1.2.8
+sbt.version=1.3.7
```

Comment:
So far we have kept everything at sbt 1.2.x levels due to some bad experiences with early versions of sbt 1.3.x. Those might have been solved in the mean time but perhaps we should tackle the migration to sbt 1.3.x as a separate PR and refresh our memories on what went wrong the first time we tried that (can't currently remember I'm afraid).

Reply:
Makes sense. I'll revert to 1.2.8.